torchao/dtypes/affine_quantized_tensor.py

```diff
@@ -36,6 +37,22 @@
 
 aten = torch.ops.aten
 
+def validate_float8_params(
+    input_float, mapping_type, target_dtype, quant_min, quant_max, eps, scale_dtype, zero_point_dtype, preserve_zero, zero_point_domain, layout_type, use_hqq
+):
+    assert input_float.is_floating_point(), "input_float must be a floating point tensor"
```

Comment:
actually based on these it seems most of the args are irrelevant for float8, maybe just splitting the op makes more sense?

Reply:
Yes, created a new op from_float_to_floatx. It'll be calling the existing from_float but with some pre-defined param values. Also for future we'll need to rename and refactor these methods.